configure.py

```diff
@@ -75,6 +75,19 @@ def main(args):
             if options.mock:
                 sources += ["linux/mockfile.c"]
         build.static_library("cpuinfo", map(build.cc, sources))
+        if build.target.is_ppc64:
+            if build.target.is_linux:
+                sources += [
+                    "powerpc/cache.c",
+                    "powerpc/uarch.c",
+                    "powerpc/linux/chipset.c",
+                    "powerpc/linux/clusters.c",
+                    "powerpc/linux/cpuinfo.c",
+                    "powerpc/linux/hwcap.c",
+                    "powerpc/linux/init.c",
+                    "powerpc/linux/ppc64-isa.c",
+                        ]
```

Comment:
Should be on the same line as `sources` if one to follow the existing pattern
```suggestion
                    ]
```

Reply:
I have removed extra space